public/components/event_analytics/explorer/save_panel/save_panel.tsx

```diff
@@ -146,18 +115,17 @@ export const SavePanel = ({
       {showOptionList && (
         <>
           <EuiFormRow display="columnCompressedSwitch">
-          <EuiToolTip
-          content="Only Time Series visualization and a query including stats/span can be saved as Metric" >
-            <EuiSwitch
-              showLabel={true}
-              label="Save as Metric"
-              checked={checked}
-              onChange={onToggleChange}
-              compressed
-              disabled = {!((isEqual(curVisId, 'line')) && spanValue)}
```

Comment:
Never mind I see that this is passed down now, but wondering if matching the regex is doing the same as this block of code: 
`useEffect(() => {
    if (isEqual(selectedContentTabId, TAB_CHART_ID)) {
      const statsTokens = queryManager.queryParser().parse(tempQuery).getStats();
      const updatedDataConfig = getDefaultVisConfig(statsTokens);
      setSpanValue(!isEqual(typeof updatedDataConfig.span, 'undefined'));
    }
  }, [tempQuery, selectedContentTabId, curVisId]);`

Reply:
yes the quick change i made is to match a span in query.